docs/conf.py

```diff
@@ -7,8 +7,17 @@
 from recommonmark.transform import AutoStructify
 from sphinx_scylladb_theme.utils import multiversion_regex_builder
 
+import os
```

Comment:
Not related to this patch, and we don't need to do it now, but I think eventually we should have a comment in the beginning of this file explaining what it does and who's supposed to run it and why.

Reply:
ah ok - sure we can add comments to the file